cv_next/app/auth/callback/route.ts

```diff
@@ -16,10 +16,13 @@ export async function GET(request: Request) {
     if (!error) {
       if (checkRedirect(next)) {
         return NextResponse.redirect(
-          `${origin}${Definitions.AUTH_DEFAULT_REDIRECT}${next}`
+          `${process.env.NEXT_PUBLIC_BASE_URL}${Definitions.AUTH_DEFAULT_REDIRECT}${next}`
         );
       } else {
-        const notFoundUrl = new URL("/not-found", origin);
+        const notFoundUrl = new URL(
+          "/not-found",
+          process.env.NEXT_PUBLIC_BASE_URL
+        );
```

Comment:
Is this change related to this PR?
Also - I recall a similar change in a different PR, please check.

Reply:
Isn't related, I'll check where it was added because I think that it was